### PR TITLE
Update/fix tabs layout

### PR DESCRIPTION
**Pull request by pjanik**
This PR fixes a few bugs found by Saravanan during QA:
https://www.pivotaltracker.com/story/show/181127423/comments/229981270
New variables should make tabs easier to work with.

### CODE REVIEW

File: css-modules/side-container.less [hunk 1]
```diff
@@ -35,7 +37,7 @@
       display: inline-block;
       position: relative;
       list-style: none;
-      margin-bottom: -1px;
+      // margin-bottom: -1px;
```
Comment (kswenson) [inline css-modules/side-container.less:40]: Do you want to keep this here or can it be removed now?

File: css-modules/side-container.less [hunk 2]
```diff
@@ -61,11 +63,12 @@
 
     .tabSelected {
       .tabInsideContainer {
-        padding-top: 13px;
+        height: @tabSelectedHeight - @tabTopPadding;
+        // padding-top: 13px;
```
Comment (kswenson) [inline css-modules/side-container.less:67]: Keep?

**Review by kswenson — APPROVED** [review]
👍 LGTM

File: css-modules/side-container.less (hunk 1 above)
Comment (pjanik) [reply css-modules/side-container.less]: Removed, thanks.

File: css-modules/side-container.less (hunk 2 above)
Comment (pjanik) [reply css-modules/side-container.less]: Removed, thanks.

**Pull request merged**